docs/pages/desktop-access/getting-started.mdx

```diff
@@ -444,16 +444,36 @@ the filepath to the `der_ca_file` configuration variable.
   order to resolve this.
 </Admonition>
 
+Install Teleport on the host where you will run the Teleport Deskop Service:
+
+(!docs/pages/includes/install-linux.mdx!)
+
 In order to enable Desktop Access in Teleport, add the following section in
-`teleport.yaml` on your Linux server. For a detailed description of these
+`/etc/teleport.yaml` on your Linux server. For a detailed description of these
 configuration fields, see the
 [configuration reference](./reference/configuration.mdx) page.
 
 <Details title="Desktop Service Configuration" min="8.0" scopeOnly={true} scope={["oss", "enterprise"]}>
```

Comment:
Should we use a `Tabs` here so users who've forgotten to pick their scope can see the other option?